target/linux/ath79/image/generic.mk

```diff
@@ -314,6 +331,27 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/belkin_f9j1108-v2
+  SOC := qca9558
+  DEVICE_VENDOR := Belkin
+  DEVICE_MODEL := F9J1108v2 (AC1750 DB Wi-Fi)
+  DEVICE_PACKAGES += kmod-kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport
+  KERNEL_SIZE := 5120k
```

Comment:
No point to have KERNEL_SIZE if it is not used in a recipe

Reply:
also you wouldn't want to set a static kernel size because it will change on sysupgrade

Reply:
OK, will remove